playground/dapr/Dapr.AppHost/Program.cs

```diff
@@ -1,7 +1,13 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
+var rmq = builder.AddRabbitMQ("rabbitMQ")
+                   .WithManagementPlugin()
+                   .WithEndpoint("tcp", e => e.Port = 5672)
+                   .WithEndpoint("management", e => e.Port = 15672);
+
 var stateStore = builder.AddDaprStateStore("statestore");
-var pubSub = builder.AddDaprPubSub("pubsub");
+var pubSub = builder.AddDaprPubSub("pubsub")
+                    .WaitFor(rmq);
 
 builder.AddProject<Projects.DaprServiceA>("servicea")
        .WithDaprSidecar()
```

Comment:
Dumb Question - would the Sidecar itself support the WaitFor in this case if the pubsub is waiting? Can the WaitFor also apply to the DaprStateStore?

Reply:
The `WaitFor` extension method applies to any resource that implements the `IResourceWithWaitSupport` interface. So in this case the state store does (through inheritence).